src/more-infos/more-info-configurator.html

```diff
@@ -129,11 +149,38 @@
     return stateObj.state === 'configure';
   }
 
+  setDefaultFieldInputs(stateObj) {
+    var values = {};
+    for (var field of this.stateObj.attributes.fields) {
```

Comment:
I'd instead do something like this, since it seems that the transpiling doesn't like this:

```javascript
const values = {};
Object.entries(stateObj).forEach(([key, value]) => {
  ...
});
return values;
```

Reply:
I didn't like the syntax of reduce. It didn't 'read' well imo. So I went back to forEach.
(everyone knows what foreach does 😄 )